frontend/src/global_styles/openproject/_scm.sass

```
@@ -73,27 +73,72 @@ li.change
 .repository--revision-toolbar
   margin-top: 3rem
 
-.repository--checkout-instructions--url
-  margin-top: 1rem
+.repository-checkout-container
+  width: 50%
+  display: flex
 
-.repository-breadcrumbs
-  font-size: 0.9rem
-  margin: 15px 0
+.repository-input-group
+  display: flex
 
-.repository-breadcrumbs--home
-  font-size: 0.75rem
+.repository-checkout-input-group--affix
+  background: var(--button-default-bgColor-rest)
+  border: 1px solid var(--button-default-borderColor-rest)
+  color: var(--body-font-color)
+  border-radius: 2px
+  white-space: nowrap
+  display: flex
+  align-items: center
+  padding: 0 5px
+  height: 36px
 
-.repository-bradcrumbs--identifier
-  display: inline-block
-  margin-left: 10px
+  &.-prepend
+    border-right: none !important
 
-.repository-breadcrumbs--sep
-  display: inline-block
-  margin: 0 2px
+  &.-append
+    border-left: none !important
 
-  &::before
-    content: '▸'
-    color: var(--fgColor-muted)
+.revisions-action-container
+  display: flex
+  gap: 0.5rem
+  width: 50%
+  .button
+    height: 36px
+
+.revisions-toolbar-container
+  display: flex
+
+.revisions-title-container
+  flex: 1 1
+  white-space: nowrap
+  max-width: 100%
+
+.revisions-items
+  display: flex
+
+  .revisions-item
+    margin-right: 0.5rem
+    display: flex
+    &:last-of-type
+      margin-right: 0
+    input
+      flex: 1
+      flex-basis: 150px
+
+    > div
+      height: 36px
+      display: flex
+      align-items: center
+      padding: 0 5px
+
+    label
+      font-size: 1rem
+    .button
+      margin-right: 0
+.repository--checkout-instructions-toggle
+  margin-left: 0.5rem
+  display: block
+  a
+    height: 36px
```

Even though repositories might not be the most important module, I'd still like to follow some basic principles for code quality, eg. not selecting based on tags. That will break so quickly or create unwanted side effects.

```suggestion
.revisions-items
  display: flex
  gap: 0.5 rem

  .revisions-item
    display: flex
    align-items: center
    
    .revisions-item--input // TODO: set this class not only on the input but also on the select fields
      flex: 1
      flex-basis: 150px

    .revisions-item--label
      padding: 0 5px

    .button
      margin-right: 0 // TODO: Replace with `button_no-margin` class on the button
      
.repository--checkout-instructions-toggle
  margin-left: 0.5rem // TODO: Replace with gap: on .repository-checkout-container 

  a
    height: 36px // TODO: might become obsolete when using the InputGroup for the checkout container
```